### PR TITLE
[MIG] stock_picking_line_sequence v9

### PR DESCRIPTION
**Pull request by SerpentCS**
Stock picking lines with sequence number
========================================

Provide a new field on stock moves, allowing to manage the orders of moves in a picking.

### CODE REVIEW

**Comment by SerpentCS** [comment]
This is migration of [stock_picking_reorder_lines v8](https://github.com/OCA/stock-logistics-workflow/tree/8.0/stock_picking_reorder_lines) and it is renamed to stock_picking_line_sequence

**Comment by SerpentCS** [comment]
There is the [PR for same module](https://github.com/OCA/stock-logistics-workflow/pull/262) but it was closed because the branch was reset!!

Though all previous comments are attended here!

@aheficent

**Review by AaronHForgeFlow — COMMENTED** [review]
I tested on runbot and it works fine :) . Can you check the travis errors?

**Review by zakiuu — CHANGES_REQUESTED** [review]
Can you please follow the migration process, to get the right history
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

File: stock_picking_line_sequence/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# Copyright 2016 Camptocamp SA
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import models, api, fields
+
+
+class StockMove(models.Model):
```
Comment (zakiuu) [inline stock_picking_line_sequence/models/stock.py]: @SerpentCS it's better to put each file in a separated file

**Comment by JayVora-SerpentCS** [comment]
@darshan-serpent please check

**Comment by serpentcs-dev1** [comment]
@zakiuu I guess it is not possible to preserve the commits as the module name has changed! What do you think?

cc @JayVora-SerpentCS

**Comment by zakiuu** [comment]
@darshan-serpent not sure I do understand what you mean, but if your PR is a migration of a module in V9 then the commits should be preserved even you've changed the name

**Comment by MiquelRForgeFlow** [comment]
Superseeded by https://github.com/OCA/stock-logistics-workflow/pull/385 to add the commit history.

**Comment by LoisRForgeFlow** [comment]
Closing as superseded.

**Pull request closed without merging**